src/assets/js/api.js

```diff
@@ -1,9 +1,15 @@
 const baseUrl = 'https://pokeapi.co/api/v2';
 
-const getPoke = async () => {
-  const getResult = await fetch(`${baseUrl}/pokemon/?offset=20&limit=9`);
+export const getPoke = async () => {
+  const getResult = await fetch(`${baseUrl}/pokemon/?offset=0&limit=9`);
   const json = await getResult.json();
   return json.results;
 };
 
-export default getPoke;
+// export const getPokeDetails = async (id) =>  {
```

Comment:
Can we delete the commented code please?

Reply:
Yes. Thanks.
done